content/en/docs/High Availability/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "High Availability"
+linkTitle: "High Availability"
+weight: 170
+description: >
+  Learn how to configure High Availability in Kyverno
+---
+
+## Configure Kyverno in HA mode
+To install Kyverno in HA using Helm Chart.
+  
+  **NOTE:** Due to some cmplexities accompanied with running 2 replicas, the recommended replica counts for HA is at least 3.
+
+``` 
+helm install kyverno kyverno/kyverno --create-namespace --set=replicaCount=3 
+```
+## How HA works in Kyverno
+This section provides details on how Kyverno handles HA scenario.
+- ### Module - Webhook Server
```

Comment:
Using third level headings is sufficient here. No need to begin with a dash as it messes with anchor links.